public/index.js

```diff
@@ -124,7 +191,12 @@ recipeButtons.forEach((button) => {
         removeElements([headline, allergies, ...recipeButtons]);
         displayElements([userWantAnotherRecipe, gptResponseElement]);
       })
-      .catch((error) => console.error("Error:", error))
+      .catch((error) => {
```

Comment:
Added the gptresponse and error messages here in the catch error. So it's only triggered when an error is caught. @josueJURE